range-requests/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
```

Comment:
`range-requests` should be reviewed as a part of https://github.com/oxidecomputer/omicron/pull/6963 -- feel free to ignore these files; there should be no major changes.

Reply:
Merged; marking this resolved